packages/components/document-card/src/document-card.js

```diff
@@ -37,14 +38,18 @@ customElementDefineHelper(
 			`;
 		}
 
+		get boldClass() {
+			return this.bold ? 'bold' : '';
+		}
+
 		render() {
 			return html`
 				<div class="document-card" dir=${this.direction}>
 					${this.fileIcon}
 					<div class="card-container">
-						<div class="card-title">${this.name}</div>
+						<div class="card-title ${this.boldClass}">${this.name}</div>
```

Comment:
class name should be used in the container div and change the style in css rule.

Reply:
yes